tools/site_and_regional/run_neon.py

```diff
@@ -368,18 +333,28 @@ def run_case(self, base_case_root, run_type, run_length, overwrite=False):
         user_mods_dirs = [os.path.join(self.cesmroot,"cime_config","usermods_dirs","NEON",self.name)]
         expect(os.path.isdir(base_case_root), "Error base case does not exist in {}".format(base_case_root))
         case_root = os.path.abspath(os.path.join(base_case_root,"..", self.name+"."+run_type))
-        if os.path.isdir(case_root) and overwrite:
-            logger.info("---- removing the existing case -------")
-            shutil.rmtree(case_root)
+        if os.path.isdir(case_root):
+            if overwrite:
+                logger.info("---- removing the existing case -------")
+                shutil.rmtree(case_root)
+            else:
+                logger.warning("Case already exists in {}, not overwritting.".format(case_root))
+                return
+
+        if run_type == "postad":
+            adcase_root = case_root.replace('.postad','.ad')
+            if not os.path.isdir(adcase_root):
+                logger.warning("postad requested but no ad case found in {}".format(adcase_root))
+                return
+
         if not os.path.isdir(case_root):
             # read_only = False should not be required here
             with Case(base_case_root, read_only=False) as basecase:
                 logger.info("---- cloning the base case in {}".format(case_root))
                 basecase.create_clone(case_root, keepexe=True, user_mods_dirs=user_mods_dirs)
 
-
         with Case(case_root, read_only=False) as case:
-            case.set_value("STOP_OPTION", "nyears")
+            case.set_value("STOP_OPTION", "ndays")
```

Comment:
do we want this as nyears for 'official' runs?

Reply:
The problem with nyears is understanding leap years.   This is still defaulting to 200 years for ad and 50 for postad just expressed in terms of days.

Reply:
again, maybe a comment to this effect would be helpful. I'm thinking of users who may be making code modifications and having to run a whole new spinup and transient case.